scripts/utilities/rescal_utilities.py

```diff
@@ -405,6 +439,110 @@ def write(self, filename):
+        directory_name_suffix = [name.replace(' ', '_') + '-' + \
+                                 os.path.basename(str(params_dict_to_add[name])) for name in names]
+        yield params_dict_to_add, '--'.join(directory_name_suffix)
+
+
+
+#### NOTE: set up for power-lab machine usage, meaning on a single PC, not a cluster
+def make_run_directories(fixed_params, variable_params, experiment_name, run_header, run_name):
+    '''Create a set of directories that contain all needed files for separate rescal runs.
+       The directories exist in a top level directory of experiment_name.
+       Each directory is names based on its varying parameters.
+       The paths to the run scripts are returned for easy execution on the
+       power-lab machines.'''
+
+    #breakpoint()
+    # make the top level directory, but don't overwrite a directory that
+    # already exists
+    experiment_directory_name = experiment_name
+    if not os.path.isdir(experiment_name):
+        os.mkdir(experiment_name)
+    # TODO deal with case that directory already exists
+    else:
+        return
+    
+    # save the paths to the run scipts for running them later
+    run_scripts = []
+
+    # create full parameter set for each run
+    # as well as the directory suffix for each run sub directory
+    for current_variable_params in all_parameter_combos(variable_params):
+        params_to_add, directory_suffix = current_variable_params
+        parameters = {**fixed_params, **params_to_add}
+        this_run = Design_a_run()
+        #this_run.set_header(run_header)
+        this_run.set_name(run_name)
+        # create a directory for each run inside the experiment directory
+        run_directory = os.path.join(experiment_directory_name, directory_suffix)
+        if not os.path.isdir(run_directory):
+            os.mkdir(run_directory)
+
+        this_run.set_directory(run_directory)
+        this_run.set_parameters(parameters)
+        this_run.write()
+
+        # store all the run scripts for future use
+        run_scripts.append(os.path.join(run_directory, run_name + '.run'))
+
+    return run_scripts
+
+
+#### NOTE: set up for power-lab machine usage, meaning on a single PC, not a cluster
+def run_rescals(run_scripts):
+    '''Takes path to a set of run_scripts that should be in the proper locations
+       to run rescal and runs each of them. Runs should be asynchronous.'''
+
+    # if any spaces in path names, turns ' ' into '\ ' so the shell can understand them
+    modded_scripts = []
+    for run_script in run_scripts:
+        modded_scripts.append(run_script.replace(' ', '\\ '))
+
+    processes = []
+    # move to the directory of each run_script and run it
+    for modded_script in modded_scripts:
+        processes.append(subprocess.Popen('cd ' + os.path.dirname(modded_script) + \
+                                          ' && ' +  modded_script, shell=True))
+    for p in processes:
```

Comment:
`check_output`

Reply:
From looking at the documentation, check_output uses stdout. I'm using a different file descriptor because I don't want to deal with all the stuff that ReSCAL sends to stdout. If there's a way to not use stdout but still get the blocking behavior of check_output I'd be willing to use it.

Reply:
Oops, I commented on the wrong comment, this was supposed to be for the Popen in data run.py.

Reply:
In this case I don't want to use check_output because it would block. It would be like calling 
`Popen(...).wait()`     
in the the for loop where I start all the processes.
I would have to wait for the each processes to finish before starting the next one.

Reply:
Could you make a comment to that effect so that it's clear to folks reading the script that it runs in parallel there?